src/NServiceBus.Persistence.ServiceFabric/StaticVersions.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace NServiceBus.Persistence.ServiceFabric
+{
+    using System;
+    using System.Diagnostics;
+    using System.Reflection;
+
+    static class StaticVersions
```

Comment:
If the only static version that we keep here is the persistence assembly version, why not to call the class `PersistenceVersion`?